pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -550,9 +559,14 @@ func (br *OVSBridge) createPort(name, ifName, ifType string, ofPortRequest int32
 		}),
 		ExternalIDs: externalIDMap,
 	}
+	var portInterface interface{}
+	portInterface = port
+	if vlanTag != nil && *vlanTag > 0 {
```

Comment:
If VLAN 0 means no VLAN (I think that is a common convention in switch configuration), why not use uint16 as the argument type, instead of *uint32?

Reply:
For OVS port configuration `AccessPort`, the Tag type is `uint32`, thus I use `uint32` for this part.
Do you think we should change all `uint32` vlanTag to `uint16` except `AccessPort`?

Reply:
I feel uint16 is better, but more importantly I am asking why use a pointer, but not just the integer value, if VLAN 0 just means no VLAN.

Reply:
OK, I'll change all vlanID from *uint32 to *uint16.
For pointer, the reason is same as https://github.com/antrea-io/antrea/pull/3247#discussion_r818366455
If we don't want to support VID=0 in future, we can use vlanID=0 to indicate nonVLAN and use `uint16` instead of `*uint16`.

Reply:
I think pointer does not make sense at least for this func, as you anyway checks: *vlanTag > 0, so I would suggest to change to uint16.

Reply:
Refactored all vlanID from `*uint16` to `uint16`